base_layer/wallet_ffi/src/callback_handler.rs

```diff
@@ -213,11 +213,11 @@ where TBackend: TransactionBackend + 'static
                                 TransactionEvent::TransactionMined(tx_id) => {
                                     self.receive_transaction_mined_event(tx_id).await;
                                 },
-                                /// Only the above variants are mapped to callbacks
+                                // Only the above variants are mapped to callbacks
                                 _ => (),
                             }
                         },
-                        Err(e) => error!(target: LOG_TARGET, "Error reading from Transaction Service event broadcast channel"),
+                        Err(_) => error!(target: LOG_TARGET, "Error reading from Transaction Service event broadcast channel"),
```

Comment:
Rather log the actual error as well here under a trace if it's going to be too detail for an error log.